codalab/worker/local_run/docker_image_manager.py

```diff
@@ -136,6 +136,32 @@ def get(self, image_spec):
         :param image_spec: Repo image_spec of docker image being requested
         :returns: A DockerAvailabilityState object with the state of the docker image
         """
+
+        def get_local_image(image_spec, success_message, failure_message):
+            """
+            Try to get the image specified by image_spec from local. 
```

Comment:
What do you mean "from local"?

Reply:
from local machine...

Reply:
@candicegjing can you update the docstring to clarify that?

Reply:
@epicfaace Updated.